README.md

```diff
@@ -38,21 +40,21 @@ Select queries and aggregations within your Playground files and translate them
  * Python 3
  * Ruby
 
-![Query Translator](resources/screenshots/query-translator.png)
+![Export to language](resources/screenshots/export-to-language.gif)
 
 ### Document Editing
 
-MongoDB for VS Code makes it extremely easy to make changes to documents in your collections. You can open documents in an editor tab, edit them and save the changes back to MongoDB.
+Editing docunments with MongoDB for VS Code is as natural as editing any file in the code editor. Open a document in an editor tab, edit it and save the changes back to MongoDB.
```

Comment:
I am wondering if we should also provide an icon and not only a shortcut to save the document 🤔

Reply:
I think we can consider that but I would not do it now. Add it to your doc with the list of potential improvements that I have on my to do list to triage and that likely Gaurab will inherit.